Swift/ReplicatorConfiguration.swift

```diff
@@ -406,17 +414,15 @@ public struct ReplicatorConfiguration {
     
     func toImpl() -> CBLReplicatorConfiguration {
         let target = self.target as! IEndpoint
-        let c = CBLReplicatorConfiguration(database: self.database._impl, target: target.toImpl())
+        var c: CBLReplicatorConfiguration!
+        
+        c = CBLReplicatorConfiguration(target: target.toImpl())
```

Comment:
Do it in one line `var c = CBLReplicatorConfiguration(target: target.toImpl())`?